docs/source/conf.py

```diff
@@ -202,3 +202,42 @@
     "traits": ("https://docs.enthought.com/traits", None),
     "traitsui": ("https://docs.enthought.com/traitsui", None),
 }
+
+
+def run_apidoc(app):
+    """
+    Hook to run automated API documentation generation via sphinx-apidoc
```

Comment:
this is a mouthful. how about 

```suggestion
    Hook to generate API documentation via sphinx-apidoc
```

Reply:
Much better. Thank you!